src/components/VersionSelect/index.tsx

```diff
@@ -18,11 +20,18 @@ class VersionSelectBase extends React.Component<PublicProps> {
     withLeftArrow: false,
   };
 
+  onChange = (event: React.FormEvent) => {
+    const { value } = event.currentTarget as HTMLSelectElement;
```

Comment:
Huh. We used to do it in Flow like this (which doesn't work):

```js
  onChange = (event: React.FormEvent<HTMLSelectElement>) => {
    const { value } = event.currentTarget;

    this.props.onChange(value);
  };
```

Reply:
Yeah, looks like react-bootstrap types are.. not easy to work with.

Reply:
I found a slightly better type.